tui-popup/Cargo.toml

```diff
@@ -22,18 +22,21 @@ rustdoc-args = ["--cfg", "docsrs"]
 default = []
 
 ## Enables processing crossterm mouse events using [`PopupState::handle_mouse_event`]
-crossterm = ["ratatui/crossterm"]
+crossterm = ["dep:crossterm"]
```

Comment:
I'm not certain that we should just use crossterm, maybe this should work the same as ratatui and allow for multiple versions of crossterm to be used so that it works wherever?